### PR TITLE
[MRG] MAINT: fix mac instructions

### PR DESCRIPTION
**Pull request by rythorpe**

### CODE REVIEW

File: installer/mac/README.md [hunk 1]
```diff
@@ -60,10 +70,10 @@ The Xcode Command Line Tools package includes utilities for compiling code from
     ```bash
     cd ${CONDA_PREFIX}
     mkdir -p etc/conda/activate.d etc/conda/deactivate.d
-    echo "export OLD_LD_LIBRARY_PATH=\$LD_LIBRARY_PATH" >> etc/conda/activate.d/env_vars.sh
-    echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:\${CONDA_PREFIX}/lib" >> etc/conda/activate.d/env_vars.sh
-    echo "export LD_LIBRARY_PATH=\$OLD_LD_LIBRARY_PATH" >> etc/conda/deactivate.d/env_vars.sh
-    echo "unset OLD_LD_LIBRARY_PATH" >> etc/conda/deactivate.d/env_vars.sh
+    echo "export OLD_DYLD_LIBRARY_PATH=\$DYLD_LIBRARY_PATH" >> etc/conda/activate.d/env_vars.sh
```
Comment (ntolley) [inline installer/mac/README.md:73]: interesting, when did this change?

Comment (rythorpe) [reply installer/mac/README.md:74]: I honestly don't know, but nobody in the workshop using the latest MacOS was able to get their installation to completely work. I just tested it on my laptop and it works with this fix though!

File: installer/mac/README.md [hunk 2]
```diff
@@ -40,11 +40,22 @@ The Xcode Command Line Tools package includes utilities for compiling code from
...
     ```
 
+## Download HNN source code
+
+- The following commands will download the hnn source code and compile HNN's mod files for NEURON. We use the directory `hnn_source_code` for consistency with all of our instructions, but any directory can be used. You can use `git` if you prefer.
+
+    ```bash
+    curl -OL https://github.com/jonescompneurolab/hnn/releases/latest/download/hnn.tar.gz
+    mkdir hnn_source_code
+    tar -x --strip-components 1 -f hnn.tar.gz -C hnn_source_code
```
Comment (rythorpe) [inline installer/mac/README.md:50]: Note that we're now having users download the source code **before** attempting to call `environment.yml` (bug).

**Comment by rythorpe** [comment]
This is an urgent fix for the workshop today. @ntolley @jasmainak please rebase and merge when ready!

**Comment by ntolley** [comment]
Thanks @rythorpe! Might be worth doing some googling to see what changed after the workshop. Seems like something on the conda end?

**Comment by rythorpe** [comment]
I think it was an operating system or NEURON thing actually.

**Comment by jasmainak** [comment]
Thanks @rythorpe for taking care of it!

**Pull request merged**